app/templates/application.hbs

```diff
@@ -3,16 +3,22 @@
     1. create common template for all pages (About, Contact, Welcome, etc...) - they will contain navbar, content and footer
     2. use different template for network (no footer, different navbar behavior
 }}
-{{elvis-nav dropMenu=dropMenu loginVisible=loginVisible}}
 
 <div class="c-notifications__container">
     {{#each notifications as |notification|}}
         {{notification-message notification=notification}}
     {{/each}}
 </div>
-{{outlet}}
 
+
+
+<div id="nav-wrapper">
```

Comment:
Why do we need a wrapper around the nav?

Reply:
the nav positioning was acting unexpectedly. probably because it inherited many styles from materialize. this seemed like more comfortable solution at the time i was adjusting the network layout. but i agree its dirty, will have a look at it.